rfc6962-bis.xml

```diff
@@ -1856,71 +1853,35 @@ value="Montreal"/>
-
-    opaque SHA256Hash[32];
-
-    struct {
-        Version version = v1;
-        SHA256Hash log_id;
-        uint64 timestamp;
-        SctExtensions extensions;
-        digitally-signed struct {
-            Version version = v1;
-            uint8 signature_type = 0;  /* "certificate_timestamp" */
-            uint64 timestamp;
-            TransType type;  /* x509_entry(0) or precert_entry(1) */
-            select (type) {
-                case x509_entry: ASN.1Cert;
-                case precert_entry: PreCert;
-            } signed_entry;
-            SctExtensions extensions;
-        } signature;
-    } SignedCertificateTimestampV1;
-
-    struct {
-        SHA256Hash log_id;
-        uint64 timestamp;
-        uint64 tree_size;
-        SHA256Hash sha256_root_hash;
-        digitally-signed struct {
-            Version version = v1;
-            uint8 signature_type = 1;  /* "tree_hash" */
-            uint64 timestamp;
-            uint64 tree_size;
-            SHA256Hash sha256_root_hash;
-        } signature;
-    } SignedTreeHeadDataV1;
-
-    struct {
-        SHA256Hash log_id;
-        uint64 tree_size_1;
-        uint64 tree_size_2;
-        SHA256Hash consistency_path&lt;1..2^8-1&gt;;
-    } ConsistencyProofDataV1;
-
-    struct {
-        SHA256Hash log_id;
-        uint64 tree_size;
-        uint64 leaf_index;
-        SHA256Hash inclusion_path&lt;1..2^8-1&gt;;
-    } InclusionProofDataV1;</artwork>
-      </figure>
+    <section title="Supporting V1 and V2 simultaneously" anchor="v1_coexistence">
+      <t>
+        Certificate Transparency logs have to be either V1 (conforming to <xref target="RFC6962"/>) or V2 (conforming to this document), as the data structures are incompatible and so a V2 log could not issue a valid V1 SCT.
+      </t>
+      <t>
+        CT clients, however, can support V1 and V2 SCTs, for the same certificate, simultaneously, as V1 SCTs are delivered in different TLS, X.509 and OCSP extensions than V2 SCTs.
+      </t>
+      <t>
+        V1 and V2 SCTs for X.509 certificates can be validated independently.
+        For precertificates, V2 SCTs should be embedded in the TBSCertificate before submission of the TBSCertificate (inside a V1 precertificate, as described in Section 3.1. of <xref target="RFC6962"/>) so that TLS clients conforming to <xref target="RFC6962"/> but not this document are oblivious to the embedded V2 SCTs.
+        An issuer can follow these steps to produce an X.509 certificate with valid V1 and V2 SCTs:
+        <list style="symbols">
+          <t>
+            Create a CMS precertificate as described in <xref target="Precertificates"/> and submit it to V2 logs.
+          </t>
+          <t>
+            Embed the obtained V2 SCTs in the TBSCertificate, as described in <xref target="cert_transinfo_extension"/>.
+          </t>
+          <t>
+            Create a V1 precertificate, as described in Section 3.1. of <xref target="RFC6962"/> and submit it to V1 logs.
```

Comment:
I don't think it's clear that "Create a V1 precertificate..." needs to use the TBSCertificate from the previous step.

How about:
"Use that TBSCertificate to create a v1 precertificate..."